### PR TITLE
Complete Join Details Model 

### PR DESCRIPTION
**Pull request by HemaSAli**
Completing the Process of join which contain other two Stages 
- First stage is about level in English
- Second stage is about job title, this stage relates to #53

### CODE REVIEW

**Comment by FarahZaqout** [comment]
Conflicts with master. Please pull it and push the changes again.

File: client/src/components/HomePage/modal/JoinModal/index.jsx [hunk 1]
```diff
@@ -4,75 +4,113 @@ import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
...
+    if (first && last && email && password) {
+      this.setState({ completeJoin: true });
+    }
   };
 
+  backFromDetails = () => {
+    this.setState({ completeJoin: false });
+  }
+
+  logState = () => {}
```
[the remark this answers is not in the document]
Comment (HemaSAli) [reply client/src/components/HomePage/modal/JoinModal/index.jsx]: At first, Thanks for your great words and feedback.
Ok at first let's talk about the function, 
let's write in it
```
console.log(this.state)
```
this function exist in the main class of Join modal, so after second Stage, if I click on next, I used this function to see what is the last state which I will send it to backEnd, It's just a test
I know it's not the time for it, so I will delete it, 
image

About BEM and Input Validation, till now I prefer to do it in other branch, or I can do it in this branch tonight, especially the BEM needs a lot of changes

[the remark this answers is not in the document]
Comment (FarahZaqout) [reply client/src/components/HomePage/modal/JoinModal/index.jsx]: Do it in this branch, and don't worry about the time.

[the remark this answers is not in the document]
Comment (HemaSAli) [reply client/src/components/HomePage/modal/JoinModal/index.jsx]: Ok, I will start now :+1:

**Pull request merged**